eogrow/pipelines/rasterize.py

```diff
@@ -100,9 +100,15 @@ def __init__(self, *args: Any, **kwargs: Any):
         super().__init__(*args, **kwargs)
 
         self.filename: Optional[str] = None
+        assert len(set([c.output_feature[0].is_temporal() for c in self.config.columns])) == 1, \
+            "All output features have to be either temporal or timeless!"
+
         if isinstance(self.config.vector_input, str):
             self.filename = self._parse_input_file(self.config.vector_input)
-            self.vector_feature = FeatureType.VECTOR_TIMELESS, f"TEMP_{uuid.uuid4().hex}"
+            feature_type = FeatureType.VECTOR_TIMELESS
+            if self.config.columns[0].output_feature[0].is_temporal():
+                feature_type = FeatureType.VECTOR
```

Comment:
I'd prefer something along the lines of:
```python
output_is_temporal = self.config.columns[0],..
feature_type = FeatureType.VECTOR if output_is_temporal else FeatureType.VECTOR_TIMELESS
 ```

Reply:
Changed.